apps/api/src/cache/cache.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable, OnModuleDestroy } from '@nestjs/common'
+import { createClient, RedisClientType } from 'redis'
+import { User } from '@prisma/client'
+
+@Injectable()
+export class CacheService implements OnModuleDestroy {
+  private readonly redisClient: RedisClientType
+  private readonly userPrefix = 'user-'
```

Comment:
Since these are static, you should rename it to `USER_PREFIX`. And also, make the variable `static`